doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -105,6 +105,7 @@ Matter
 
   * The :ref:`ug_matter_device_low_power_configuration` page with the information about Intermittently Connected Devices (ICD) configuration.
   * The Factory Data Guide regarding the changes for the SPAKE2+ verifier, and generation of the Unique ID for Rotating Device ID.
+  * The Bridge Application is no more experimental.
```

Comment:
Should it be here or under Matter samples - or in both places? @wiba-nordic

Reply:
Good catch. It should actually be in the Applications section where the Matter Bridge application itself resides. 
I would suggest adding it as the first bullet before the Added bullet with the following text:
```
* The Matter Bridge application is now supported instead of experimental.
```